docs/elements/hx-tab/index.html

```diff
@@ -0,0 +1,51 @@
+---
+title: <hx-tab>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="attributes">Attributes</h2>
+  <dl>
+    <dt>
+      aria-controls
```

Comment:
Sorry, I missed these. Should the attributes in this column all be in code blocks?

Reply:
So `<code>aria-controls</code>`

Reply:
all `<dt>` values are bolded instead of `<code>` -- to address comments :)